server/database/seed.js

```diff
@@ -0,0 +1,91 @@
+const faker = require('faker');
+const mysql = require('mysql2');
+const Sequelize = require('sequelize');
+
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'mikamaus'
+})
+
+con.connect(function(err) {
+  if (err) {
+    console.error('error connecting: ' + err.stack);
+    return;
+  }
+  console.log('connected to mysql');
+  con.query('CREATE DATABASE IF NOT EXISTS `etsymary`')
+})
+
+const connection = new Sequelize('etsymary', 'root', 'mikamaus', {

```

Comment:
I recommend, making a config file for this. When we get to building our proxy server, we will have to change the username and password. (Not sure about the username, I think we all are on "root")

You could also leave it as it is, but mention in your README.md to change the password in line 20.